bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -252,19 +250,20 @@ private func dummy() {
     _ = NSLocalizedString("Select a color", comment: "")
     _ = NSLocalizedString("Select a color above", comment: "")
     _ = NSLocalizedString("Select a layout", comment: "")
-    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select font size option value e.g: \"Selected: Large\".\ntranslators: %s: Select control option value e.g: \"Auto, 25%\".")
```

Comment:
- Modification
- [Code reference 1](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.native.js#L141:L141)
- [Code reference 2](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L93:L93)

This string is used in two places and a different comment is provided from each one, for this reason, the value of the comment property contains two lines.